skycoin/dmsg
# Updated makefile and terminal import

Changes:	
- updated `github.com/incu6us/goimports-reviser` to `github.com/incu6us/goimports-reviser/v2` in Makefile : install-linters
- updated the depreciated library `golang.org/x/crypto/ssh/terminal` to `golang.org/x/term` in `dmsgpty/cli.go`

How to test this PR:
1. Run `make install-linters`
2. Run `make format && make check`
